### ISSUE TITLE
Make "outline color" easier to identify

### ISSUE THREAD

**Issue by pgilfernandez** [synfig/synfig#2893]
**Is your feature request related to a problem? Please describe.**
When selecting colors, it's hard to remember what color is the "fill" and which one is the "outline" one.

**Describe the solution you'd like**
I would suggest to do as other softwares do:

![fore-background_colors](https://user-images.githubusercontent.com/5942369/202755926-48ad7cfb-c520-45dd-849a-aba3c574fc14.jpg)

Thanks

**Describe alternatives you've considered**
none

**Additional comments**
none

**Comment by yoSachinkr** [comment]
There's already a distinction that the outline color is at the top left corner while the fill color is at the bottom right corner. And in my opinion, area which represent color should appear completely filled for better visibility.

**Comment by Keyikedalube** [comment]
Or what about tooltips like GIMP 👇🏽👇🏽👇🏽

![Screenshot from 2022-12-02 20-24-17](https://user-images.githubusercontent.com/24214509/205321333-9e15edcb-0fb4-43bb-8390-6cc5d9263a3b.png)

**Comment by veermetri05** [comment]
It already does show tooltip on Outline Color and Fill Color
![image](https://user-images.githubusercontent.com/37113369/209795643-6d10abac-5952-466d-bee6-c2d93f805c31.png)

**Comment by pgilfernandez** [comment]
Yes, yes, but... my point is that it is not undoubtedly easy to see them at a fist sight. You have to either have learned their positions, check the preview to their right, read the tooltips or just try and check...

With a simple visual helper it would be impossible to fail, no matter if you are newbie, a pro coming from another software or a Synfig guru with a bad day, hehehe.

Anyway, it is just an UX-UI suggestion, if you all feel like it is fine as it is just forgive me =)

Cheers
